Content/ReleaseBriefings/4_6/Section_GeneralEnhancements.tex

```diff
@@ -0,0 +1,154 @@
+    \item Fix a warning when a user calls the cmake function \texttt{kokkos\_check} from a \texttt{<PackageName>Config.cmake} file
+    {\tiny \begin{verbatim}
+CMake Warning (dev) at /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
+  The package name passed to `find_package_handle_standard_args`
+  (Kokkos_DEVICES) does not match the name of the calling package (SomePackage).
+  This can lead to problems in calling code that expects `find_package`
+  result variables (e.g., `_FOUND`) to follow a certain pattern.
+Call Stack (most recent call first):
+  ... /kokkos/lib/cmake/Kokkos/KokkosConfigCommon.cmake:110 (find_package_handle_standard_args)
+  ...
+This warning is for project developers.  Use -Wno-dev to suppress it.
+    \end{verbatim}}
+ \end{itemize}
+\end{frame}
+
+\begin{frame}[fragile]{\texttt{inclusive\_scan} performance improvements}
+  With the Cuda and HIP backends, \texttt{Kokkos::Experimental::inclusive\_scan} now calls the vendor versions in Thrust
+  \begin{itemize}
+     \item The vendor versions are faster than the \texttt{Kokkos::parallel\_scan}-based default implementation
+     \item Thrust requires \texttt{Kokkos\_ENABLE\_ROCTHRUST} to be \texttt{ON} (which is the default)
+  \end{itemize}
+\end{frame}
+
+\begin{frame}[fragile]{Reduce tooling interface overhead}
+  Reduced the overhead of Kokkos tools related checks
+  \begin{itemize}
+      \item Store the information whether Kokkos tools are enabled after each modification to the tools' callbacks
+      \item Previously, this value was recomputed for every event (\texttt{parallel\_for}, \texttt{fence}, etc.)
+      \item Most noticeable in small serial kernels (around 100 elements)
+  \end{itemize}
+\end{frame}
+
+%==========================================================================
+\begin{frame}[fragile]{SIMD reductions and compound assignments}
+  Added reductions and remaining compound assignments to Kokkos SIMD
+
+  \begin{itemize}
+    \item \texttt{basic\_simd\& operator/=(basic\_simd\&, U\&\&)}
+    \item \texttt{basic\_simd\& operator>>=(basic\_simd\&, U\&\&)}
+    \item \texttt{basic\_simd\& operator<<=(basic\_simd\&, U\&\&)}
+  \end{itemize}
+
+  \vspace{5pt}
+  
+  \begin{itemize}
+    \item \texttt{T reduce\_min(const basic\_simd\& x)}
+    \item \texttt{T reduce\_max(const basic\_simd\& x)}
+    \item \texttt{T reduce(const basic\_simd\& x, const mask\_type\& mask, T identity\_element, BinaryOperation binary\_op)}
+      \begin{itemize}
+        \item Supported binary operations are: std::plus, std::multiplies, std::bit\_and, std::bit\_or and std::bit\_xor
+        \item std::plus is used if binary op is not specified
+      \end{itemize}
+  \end{itemize}
+
+\end{frame}
+%==========================================================================
+\begin{frame}[fragile]{Avoid storing a view in RandomAccessIterator}
```

Comment:
RandomAccessIterator is in `Impl::` so we will need to find some other way of explaining that sort got faster

Reply:
Same comment about including performance numbers

Reply:
Using the same benchmark described in https://github.com/kokkos/kokkos/pull/7304, I just quickly tested it for `Kokkos::reverse_copy` and `Kokkos::find`, but I could not observe any notable changes in runtime before and after https://github.com/kokkos/kokkos/pull/7304.
Fixes to Kokkos sort runtime performance was already covered during last release briefing, so I'm not sure if it should be mentioned again.

Reply:
@ldh4 on what was that calculated?

Reply:
@JBludau Host only build with gcc on Skylake.
I think it's going to be best to communicate during the briefing that this change in `RandomAccessIterator` does not necessarily improve runtime all algorithms like it shows in search.

Reply:
I added a hint in https://github.com/kokkos/kokkos-tutorials/pull/125/commits/a66044d5ead770f6f8056955603ea7366b2ea8b0